docs/babel.rst

```diff
@@ -0,0 +1,64 @@
+Translations
+============
+One key goal of `pyhf` is to provide seamless translations from one statistical framework to us. This page details the various ways to translate from a tool you might already be using as part of an existing analysis to `pyhf`. Many of these solutions involve extracting out the `HistFactory` workspace and then running `pyhf xml2json <cli.html#pyhf-xml2json>`_ which provides a single JSON workspace that can be loaded directly into `pyhf`.
+
+HistFitter
+----------
+
+In order to go from `HistFitter` to `pyhf`, the first step is to extract out the `HistFactory` workspaces. Assuming you have an existing configuration file, ``config.py``, you likely run an exclusion fit like so:
+
+.. code:: bash
+
+  HistFitter.py -f -D "before,after,corrMatrix" -F excl config.py
+
+The name of output workspace files depends on four parameters you define in your ``config.py``:
+
+- ``analysisName`` is from ``configMgr.analysisName``,
+- ``prefix`` is defined in ``configMgr.addFitConfig({prefix})``, and
+- ``measurementName`` is the first measurement you define via ``fitConfig.addMeasurement(name={measurementName},...)``
+- ``channelName`` are the names of channels you define via ``fitConfig.addChannel("cuts", [{channelName}], ...)``
+- ``cachePath`` is where `HistFitter` stores the cached histograms, defined by ``configMgr.histCacheFile`` which defaults to ``data/{analysisName}.root``
+
+To dump the HistFactory workspace, you will modify the above to skip the fit ``-f`` and plotting ``-D`` so you end up with
+
+.. code:: bash
+
+  HistFitter.py -wx -F excl config.py
+
+The ``-x`` flag tells `HistFitter` to dump the XML files into ``config/{analysisName}/``, with the top-level file being ``{prefix}.xml`` and all other files being ``{prefix}_{channelName}_cuts.xml``. The ``-w`` flag tells `HistFitter` to (re)create the `HistFactory` workspace stored in ``results/{analysisName}/{prefix}_combined_{measurementName}.root``.
+
+Typically, ``prefix = 'FitConfig'`` and ``measurementName = 'NormalMeasurement'``. For example, if the following exists in your ``config.py``
+
+.. code:: python
+
+  from configManager import configMgr
+  # ...
+  configMgr.analysisName = '3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg'
+  configMgr.histCacheFile = 'cache/{0:s}.root'.format(configMgr.analysisName)
+  # ...
+  fitConfig = configMgr.addFitConfig("Excl")
+  # ...
+  channel = fitConfig.addChannel("cuts", ['SR_0L'], 1, 0.5, 1.5)
+  # ...
+  meas1=fitConfig.addMeasurement(name="DefaultMeasurement",lumi=1.0,lumiErr=0.029)
+  meas1.addPOI("mu_SIG1")
+  # ...
+  meas2=fitConfig.addMeasurement(name="DefaultMeasurement",lumi=1.0,lumiErr=0.029)
+  meas2.addPOI("mu_SIG2")
+
+Then, you expect the following files to be made:
```

Comment:
Maybe explicitly state what these files are?

Reply:
Not in scope. This is for people using HistFitter. Not for random users.